mods/ra2/rules/allied-structures.yaml

```diff
@@ -283,7 +283,7 @@ gaairc:
 		Factions: korea
 	ProvidesPrerequisite@default:
 	Building:
-		Footprint: xxx xxx
+		Footprint: x=x x==
```

Comment:
Why this footprint in particular?

Reply:
The first `x´ is the pad that comes with the main building. The 3 `=` are needed so another actor can be placed on top of it providing 1 more pad each.